activerecord/test/cases/migration_test.rb

```diff
@@ -1347,6 +1347,22 @@ def test_changing_columns
       assert_equal "This is a comment", column(:birthdate).comment
     end
 
+    if supports_text_column_with_default?
+      def test_default_functions_on_columns
+        with_bulk_change_table do |t|
+          t.string :name, default: -> { "UUID()" }
```

Comment:
This fails on postgress, you'll have to use another function for it, or scope this test to only run against MySQL. There's a `current_adapter?` helper for that.

Reply:
I made the tests use the correct function on Postgres instead using `current_adapter?` as suggested :-) Thanks!

Reply:
The reamaining CI failure look legit.

Reply:
Oh - it was green here a minute ago :-O

Reply:
I fixed the issue. A table query was made if the default was a number, which we now detect and avoid :-)